schema/partner_show.js

```diff
@@ -265,6 +263,8 @@ const PartnerShow = {
         return show;
       });
   },
+  // ObjectIdentification
+  isType: (obj) => obj.partner !== undefined && obj.display_on_partner_profile !== undefined,
```

Comment:
I agree, it feels… meh. The most fool-proof solution would be to add a `type` field to the Gravity response, but that also feels… meh ¯\_(ツ)_/¯

Today I learned of [`GraphQLObjectType.isTypeOf `](http://graphql.org/docs/api-reference-type-system/#graphqlobjecttype), not sure what it’s for yet, but if similar in purpose I might at least use that in the future, instead of introducing my own API.